backend/src/nodes/properties/expression.py

```diff
@@ -199,3 +199,15 @@ def Image(
         fields["width"] = field(size_as, "width")
         fields["height"] = field(size_as, "height")
     return named("Image", fields)
+
+
+def PyTorchModel(
```

Comment:
I should probably get rid of this since I don't use it at all, yeah?

Reply:
If you don't use it, then yes please. I only added `Image` because I used it a lot before Navi parsing. `Image` is still used a bit, but I also contemplated removing it.